src/pkgTypes.ts

```diff
@@ -0,0 +1,97 @@
+import type { RawPkg, StyleType } from './@types/pkg';
+import type { File } from './jsDelivr';
+import { datadog } from './utils/datadog';
+
+const styleFileExtensions: StyleType[] = ['css', 'less', 'scss'];
+const jsFileExtensions = ['js', 'mjs', 'cjs'];
```

Comment:
maybe what we should be looking for is all extensions that aren't in test, documentation or examples. That can then be used frontend for a filter without being opinionated about what's a script and what's a style